sdk/src/Core/AWSConfigs.cs

```diff
@@ -100,6 +102,27 @@ public static partial class AWSConfigs
 
         #region Clock Skew
 
+        // Tests can override this DateTime source.
+        internal static Func<DateTime> utcNowSource = () => DateTime.UtcNow;
```

Comment:
Move this to line 77.
Instead of constructing a lambda here, please add a new method `GetUtcNow`, and set `utcNowSource = GetUtcNow;`